src/allencell_ml_segmenter/thresholding/thresholding_service.py

```diff
@@ -162,10 +162,10 @@ def _save_thresh_image(
             OmeTiffWriter.save(image, str(new_image_path))
 
     def _threshold_image(self, image: np.ndarray) -> np.ndarray:
-        threshold_value: float = (
+        threshold_value: int = (
             self._thresholding_model.get_thresholding_value()
         )
-        return (image > threshold_value).astype(int)
+        return (image > threshold_value).astype(np.uint8)
```

Comment:
the result is binary, so converting from int 32 or 64 to int 8 doesnt result in any information loss, correct?